pallets/subspace/tests/delegate_staking.rs

```diff
@@ -94,11 +95,11 @@ fn test_ownership_ratio() {
             .sum::<u64>();
         let total_tokens_before =
             total_balance + total_stake + total_delegate_stake + total_delegate_balance;
-        println!("total_tokens_before: {:?}", total_tokens_before);
+        info!("total_tokens_before: {:?}", total_tokens_before);
 
-        println!("delegate_balances before: {:?}", delegate_balances_before);
-        println!("delegate_stakes before: {:?}", delegate_stakes_before);
-        println!(
+        info!("delegate_balances before: {:?}", delegate_balances_before);
+        info!("delegate_stakes before: {:?}", delegate_stakes_before);
```

Comment:
Do the same everywhere else

```suggestion
        info!("delegate_balances before: {delegate_balances_before:?}");
        info!("delegate_stakes before: {delegate_stakes_before:?}");
```